src/js/components/Attribute.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from "react-redux"
+import "../../styles/attributes.scss";
+
+@connect((store) => {
+    return {
+        lastStep: store.nextStep
+    };
+})
+export default class Attribute extends Component {
+    render() {
+        if ('ability' === this.props.type) {
```

Comment:
Read about conditional rendering- it's awesome!

https://facebook.github.io/react/docs/conditional-rendering.html